CHANGELOG.md

```diff
@@ -167,6 +167,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Replace the `radio` module with peripheral singleton structs (#852)
 - The SPI traits are no longer re-exported in the main prelude, but from preludes in `spi::master`/`spi::slave` instead (#860)
 - The `embedded-hal-1` and `embedded-hal-async` traits are no longer re-exported in the prelude (#860)
+- Unify the low-power peripheral names (`RTC_CNTL` and `LP_CLKRST` to `LPWR`) (#1064)
```

Comment:
Not sure what's going on here, but this is in an old section from a few versions ago. Can you please move it up to the `Unreleased` section?

Reply:
Fixed